mne/io/base.py

```diff
@@ -1624,34 +1625,39 @@ def load_bad_channels(self, bad_file=None, force=False):
 
         Parameters
         ----------
-        bad_file : str
-            File name of the text file containing bad channels
-            If bad_file = None, bad channels are cleared, but this
-            is more easily done directly as raw.info['bads'] = [].
+        bad_file : path-like | None
+            File name of the text file containing bad channels.
+            If ``None`` (default), bad channels are cleared, but this
+            is more easily done directly with ``raw.info['bads'] = []``.
         force : bool
             Whether or not to force bad channel marking (of those
             that exist) if channels are not found, instead of
-            raising an error.
+            raising an error. Defaults to ``False``.
+        %(verbose)s
         """
+        prev_bads = self.info['bads']
+        new_bads = []
         if bad_file is not None:
             # Check to make sure bad channels are there
             names = frozenset(self.info['ch_names'])
             with open(bad_file) as fid:
                 bad_names = [line for line in fid.read().splitlines() if line]
-            names_there = [ci for ci in bad_names if ci in names]
-            count_diff = len(bad_names) - len(names_there)
+            new_bads = [ci for ci in bad_names if ci in names]
+            count_diff = len(bad_names) - len(new_bads)
 
             if count_diff > 0:
+                msg = (f'{count_diff} bad channels from:'
+                       f'\n{bad_file}\nnot found in:\n{self.filenames[0]}')
                 if not force:
-                    raise ValueError('Bad channels from:\n%s\n not found '
-                                     'in:\n%s' % (bad_file,
-                                                  self.filenames[0]))
+                    raise ValueError(msg)
                 else:
-                    warn('%d bad channels from:\n%s\nnot found in:\n%s'
-                         % (count_diff, bad_file, self.filenames[0]))
-            self.info['bads'] = names_there
+                    logger.warning(msg)
```

Comment:
This should be reverted, it was already using our internal mne.utils.warn. Why did you need to change this?

Reply:
You are right, this was an error on my side ... I assumed this was `warnings.warn` ... but it was `mne.utils.warn`